syncd/Syncd.cpp

```diff
@@ -797,6 +797,242 @@ sai_status_t Syncd::processBulkQuadEventInInitViewMode(
     }
 }
 
+sai_status_t Syncd::processBulkEntryWithSaiBulkSupport(
```

Comment:
this function is too big for my taste, you can split to smaller functions to better digest this, we probably would like to have functions that fit in 1-2 screens